modules/tinymce/src/plugins/lists/main/ts/core/Util.ts

```diff
@@ -1,25 +1,47 @@
+import { Toolbar, Menu } from '@ephox/bridge';
 import { Arr } from '@ephox/katamari';
 
 import Editor from 'tinymce/core/api/Editor';
+import { NodeChangeEvent } from 'tinymce/core/api/EventTypes';
 
 import * as NodeType from './NodeType';
 
 export const isCustomList = (list: HTMLElement): boolean =>
   /\btox\-/.test(list.className);
 
-export const listState = (editor: Editor, listName: string, activate: (active: boolean) => void): () => void => {
-  const nodeChangeHandler = (e: { parents: Node[] }) => {
-    const inList = Arr.findUntil(e.parents, NodeType.isListNode, NodeType.isTableCellNode)
-      .filter((list: HTMLElement) => list.nodeName === listName && !isCustomList(list))
-      .isSome();
-    activate(inList);
-  };
+const inList = (parents: Node[], listName: string): boolean =>
+  Arr.findUntil(parents, NodeType.isListNode, NodeType.isTableCellNode)
+    .filter((list: HTMLElement) => list.nodeName === listName && !isCustomList(list))
+    .isSome();
```

Comment:
```suggestion
    .exists((list) => list.nodeName === listName && !isCustomList(list));
```
I know this was like this previously, but doing a `filter` and `isSome` is the same as just doing `exists`. Feel free to ignore this though, but I just thought while we're changing this it might be worth it to fix this.

Reply:
Will do